app/case/views.py

```diff
@@ -11,6 +11,9 @@ def __init__(self, cases):
 class Start(Template):
     pageTitle = "View your case list"
 
```

Comment:
I have flake8 installed in my ide and it didn't pick this one up but will check it